### ISSUE TITLE
Change the loss function to a combo loss

### ISSUE THREAD

**Issue by jammy270** [CBICA/DeepMRSeg#24]
Modify the loss function from equally weighted losses to a combo loss:

total_loss = \alpha\ * focalIOUloss + (1-\alpha\) * ( MAEloss + BCEloss )

**Comment by ashishsingh18** [comment]
Done.
